ui/apps/everest/src/pages/databases/lastBackup/LastBackup.tsx

```diff
@@ -0,0 +1,35 @@
+import { useDbBackups } from 'hooks/api/backups/useBackups';
+import { LastBackupProps } from './LastBackup.types';
+import { BackupStatus } from 'shared-types/backups.types';
+import { IconButton, Tooltip, Typography } from '@mui/material';
+import { Messages } from '../dbClusterView.messages';
+import { WarningIcon } from '@percona/ui-lib';
+import { getLastBackupTimeDiff } from '../DbClusterView.utils';
+
+export const LastBackup = ({ dbName, namespace }: LastBackupProps) => {
+  const { data: backups = [] } = useDbBackups(dbName!, namespace, {
+    enabled: !!dbName,
+    refetchInterval: 10 * 1000,
+  });
+
+  const lastBackup = backups?.pop();
```

Comment:
sorry, missed this, do we have any effect on backups outside of this component?

Reply:
I changed it so that we retrieve the last backup by `backups[backups.length - 1]`. Thank you for pointing it out!